src/main/lrsql/input/actor.clj

```diff
@@ -61,6 +61,17 @@
    :actor-ifi    actor-ifi
    :actor-type   actor-type})
 
+
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Actor Delete
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+#_(s/fdef delete-actor-input
```

Comment:
?

Reply:
what *is* the play here? any munging necessary has been done, but I wanted to adhere to the input pattern.  Can remove this link in the chain if you think it better

Reply:
Well what I was pointing out is it's commented out, we just want a valid spec here to wrap the input normalization. I would bet the spec is a little different for it to be valid, i.e. something like:
```
:args  (s/cat :actor-ifi :lrsql.spec.actor/actor-ifi])
  :ret   (s/keys :req-un [:lrsql.spec.actor/actor-ifi])
  ```

Reply:
fixed (first arg is a one-key map, return value is the same)